src/api-calls.js

```diff
@@ -0,0 +1,24 @@
+export const getJobs = () => {
+  return fetch('https://spacetruckersapi.herokuapp.com/api/v1/jobs')
+  .then(response => response.json())
+}
+
+export const deleteJob = (id) => {
+  return fetch(`https://spacetruckersapi.herokuapp.com/api/v1/jobs/${id}`, {
+    method: 'DELETE',
+  })
+  .then(response => response.json())
+}
+
+export const bookJob = (id) => {
+  return fetch(`https://spacetruckersapi.herokuapp.com/api/v1/jobs/${id}`, {
+    method: 'PATCH',
+    body: JSON.stringify({
+      isBooked: true
+    }),
```

Comment:
These all look great, I think we can build off our existing error handling and add error handling for 300 - 400 errors.